src/Client.php

```diff
@@ -871,10 +816,7 @@ private function mapFieldsForMultipleContacts(array $data): array
             $data,
             [
                 'contacts' => array_map(
-                    [
-                        $this,
-                        'mapFieldsToIds',
-                    ],
+                    $this->mapFieldsToIds(...),
```

Comment:
Thought I was passed the point of finding new PHP syntax I didn't know, but apparently not. Have double checked, these are definitely equivalent.